core/serialization/exceptions.ts

```diff
@@ -67,6 +67,10 @@ export class MissingConnection extends DeserializationError {
    *     'IF0', or 'next'.
    * @param block The block missing the connection.
    * @param state The state object containing the bad connection.
+   * @internal
+   * @internal
+   * @internal
+   * @internal
```

Comment:
I think you might have been a little overzealous here.  (Were some of these meant to be attached to other methods?)

Reply:
Good catch! They were meant to be attached to the constructors of the other exceptions. Fixed, and did a quick search to make sure this didn't happen anywhere else.